### ISSUE TITLE
StdMsg and Scriver: add status text to window caption

### ISSUE THREAD

**Issue by dartraiden** [miranda-ng/miranda-ng#1354]
After removing "Use the contact's status icon as the window icon" option, blind users cannot see text information about the status of the contact.

**Comment by jawhien** [comment]
It seems that nothing has changed in SCRIVER :(

**Comment by dartraiden** [comment]
We just reimplement this settings again in future.

**Comment by jawhien** [comment]
Yes, it works :). I did not immediately understand why this edit field is needed, because Screenreader gets its name as "lines". Now I've set up and everything works. Thanks a lot for your help :)
